d-sgov/d-sgov-vfr-1.8.0-glosář.ttl

```diff
@@ -710,7 +712,7 @@ d-sgov-vfr-1.8.0-pojem:nesprávné-údaje-ulice
 d-sgov-vfr-1.8.0-pojem:znak-čísla-orientačního-adresního-místa
         a       <http://www.w3.org/2004/02/skos/core#Concept> ;
         <http://www.w3.org/2004/02/skos/core#broader>
-                <https://slovník.gov.cz/základní/pojem/vlastnost> , <https://slovník.gov.cz/základní/pojem/typ-vlastnosti> ;
+                <http://slovník.gov.cz/dokumentový/ofn-adresy-1.0.1/pojem/znak-čísla-orientačního> , <https://slovník.gov.cz/základní/pojem/typ-vlastnosti> ;
```

Comment:
všechny SSP URI by měly být https://

Reply:
To se mi úplně nezdá. Dokonce si myslím, že mi naopak Martin říkal, že všechny musí být http://. V ssp jsem dokonce nenašel žádný slovník, který by měl https://.

Reply:
@martinnec ?

Reply:
https v pr